driver/razermouse_driver.c

```diff
@@ -443,6 +443,23 @@ static ssize_t razer_attr_write_mode_static(struct device *dev, struct device_at
             report.transaction_id.id = 0xff;
             break;
 
+        case USB_DEVICE_ID_RAZER_NAGA_TRINITY:
+            ;
```

Comment:
Stray semicolon

Reply:
At some point the code wouldn't compile without that semicolon but it doesn't seem to be an issue now.